proto/ddl_service.proto

```diff
@@ -86,6 +86,8 @@ message CreateSinkRequest {
   stream_plan.StreamFragmentGraph fragment_graph = 2;
   // It is used to provide a replace plan for the downstream table in `create sink into table` requests.
   optional ReplaceTablePlan affected_table_change = 3;
+  // The list of object IDs that this materialized view depends on.
+  repeated uint32 dependencies = 4;
```

Comment:
Is it accurately _direct_ dependencies? I suppose the meta service can recursively resolve all dependencies.